components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SessionDataStore.java

```diff
@@ -317,35 +331,148 @@ public void clearSessionData(String key, String type) {
         }
     }
 
-    public void removeExpiredSessionData() {
+    /**
+     * This is to get the DB specific query for the session data removal, this may be overridden by the configuration
```

Comment:
Shall we change the method description to something starting like "returns db specific query ..." and ad full stops to method defentions